### ISSUE TITLE
Latex command broke

### ISSUE THREAD

**Issue by ToxicMushroom** [Melijn/melijn-bot#57]
```
2023-08-26 13:11:36 ERROR        c.k.k.e.commands.chat.ChatCommand | Error during execution of latex command (MessageReceivedEvent)
org.scilab.forge.jlatexmath.XMLResourceParseException: DefaultTeXFont.xml: error reading font 'fonts/latin/jlm_cmr10.ttf'. Error message: Problem reading font data.
	at org.scilab.forge.jlatexmath.DefaultTeXFontParser.createFont(DefaultTeXFontParser.java:393)
	at org.scilab.forge.jlatexmath.DefaultTeXFontParser.createFont(DefaultTeXFontParser.java:363)
	at org.scilab.forge.jlatexmath.FontInfo.getFont(FontInfo.java:309)
	at org.scilab.forge.jlatexmath.DefaultTeXFont.getChar(DefaultTeXFont.java:324)
	at org.scilab.forge.jlatexmath.DefaultTeXFont.getChar(DefaultTeXFont.java:284)
	at org.scilab.forge.jlatexmath.DefaultTeXFont.getChar(DefaultTeXFont.java:292)
	at org.scilab.forge.jlatexmath.DefaultTeXFont.getDefaultChar(DefaultTeXFont.java:341)
	at org.scilab.forge.jlatexmath.CharAtom.getChar(CharAtom.java:116)
	at org.scilab.forge.jlatexmath.CharAtom.createBox(CharAtom.java:91)
	at org.scilab.forge.jlatexmath.TeXFormula.createBox(TeXFormula.java:582)
	at org.scilab.forge.jlatexmath.TeXFormula.access$100(TeXFormula.java:94)
	at org.scilab.forge.jlatexmath.TeXFormula$TeXIconBuilder.build(TeXFormula.java:762)
	at org.scilab.forge.jlatexmath.TeXFormula.createTeXIcon(TeXFormula.java:796)
	at org.scilab.forge.jlatexmath.TeXFormula.createBufferedImage(TeXFormula.java:901)
	at me.melijn.bot.commands.MathExtension$setup$14$1.invokeSuspend(MathExtension.kt:197)
	at me.melijn.bot.commands.MathExtension$setup$14$1.invoke(MathExtension.kt)
	at me.melijn.bot.commands.MathExtension$setup$14$1.invoke(MathExtension.kt)
	at com.kotlindiscord.kord.extensions.commands.chat.ChatCommand$call$2.invokeSuspend(ChatCommand.kt:467)
	at com.kotlindiscord.kord.extensions.commands.chat.ChatCommand$call$2.invoke(ChatCommand.kt)
	at com.kotlindiscord.kord.extensions.commands.chat.ChatCommand$call$2.invoke(ChatCommand.kt)
	at com.kotlindiscord.kord.extensions.types.Lockable$DefaultImpls.withLock(Lockable.kt:24)
	at com.kotlindiscord.kord.extensions.commands.Command.withLock(Command.kt:46)
	at com.kotlindiscord.kord.extensions.commands.chat.ChatCommand.call$suspendImpl(ChatCommand.kt:385)
	at com.kotlindiscord.kord.extensions.commands.chat.ChatCommand.call(ChatCommand.kt)
	at com.kotlindiscord.kord.extensions.commands.chat.ChatCommand.call$default(ChatCommand.kt:378)
	at com.kotlindiscord.kord.extensions.commands.chat.ChatCommandRegistry.handleEvent$suspendImpl(ChatCommandRegistry.kt:229)
	at com.kotlindiscord.kord.extensions.commands.chat.ChatCommandRegistry$handleEvent$1.invokeSuspend(ChatCommandRegistry.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
```

**Comment by pajamaman18** [comment]
file `/melijn/org/scilab/forge/jlatexmath/fonts/jlm_cmr10.ttf` is not added to the shadowjar.

**Comment by ToxicMushroom** [comment]
fixed by [3895b0a](https://github.com/Melijn/melijn-bot/commit/3895b0a3f107c557c92179d8f865351faa2bb4fd)
